src/utilities.cpp

```diff
@@ -67,6 +66,13 @@ std::string convertIntToString(int value)
     return strs.str();
 }
 
+std::string convertSizeTToString(size_t value)
```

Comment:
Hmm... this is inconsistent with what we have avbove (i.e. `convertToDouble()`). If anything, I would expect this method to be called `convertToString()`.